### PR TITLE
Arquillian container Weld release!

### PR DESCRIPTION
**Pull request by tremes**

### CODE REVIEW

File: news/2016-07-21-arq-container-weld-releasel.asciidoc [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+layout: news
+title: Arquillian container Weld 2.0.0.Beta1 released!
+author: Tomas Remes
+priority: 1
+change_frequency: weekly
+tags: [arquillian, testing]
+---
+
+We are pleased to announce a new release of Arquillian container Weld.
```
Comment (mkouba) [inline news/2016-07-21-arq-container-weld-releasel.asciidoc]: What about `..release of embedded Arquillian container adapter for Weld`?

File: news/2016-07-21-arq-container-weld-releasel.asciidoc [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+This embedded container has deserved some love for really long time.
+We tried to refactor it and introduced new master branch which now corresponds to 2.0.0.x version stream.
```
Comment (mkouba) [inline news/2016-07-21-arq-container-weld-releasel.asciidoc]: Maybe we should also mention the existence of 1.0 branch?

File: news/2016-07-21-arq-container-weld-releasel.asciidoc [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+There is no more any `arquillian-weld-ee-embedded-1.1` or `arquillian-weld-se-embedded`.
```
Comment (mkouba) [inline news/2016-07-21-arq-container-weld-releasel.asciidoc]: I would add xml snippet with the new version.

File: news/2016-07-21-arq-container-weld-releasel.asciidoc (hunk 1 above)
Comment (tremes) [inline news/2016-07-21-arq-container-weld-releasel.asciidoc]: Yes sounds better.

File: news/2016-07-21-arq-container-weld-releasel.asciidoc (hunk 2 above)
Comment (tremes) [inline news/2016-07-21-arq-container-weld-releasel.asciidoc]: Hmmm maybe...:-)

**Comment by mkouba** [comment]
Merged, thanks!

**Pull request closed without merging**